docs/quick_start.rst

```diff
@@ -84,135 +113,248 @@ on multiple inputs.
+        comparisons          swaps      time
+        experiment                                          
+        bubble-sort        499500.0  253437.333333  0.053750
+        insertion-sort     241891.0  257609.000000  0.027219
 
 .. tip::
     Simexpal supports autocomplete via `argcomplete <https://pypi.org/project/argcomplete/>`_.
     To enable autocomplete, install ``argcomplete`` and enable global completion:
 
     .. code-block:: bash
-
+      
       $ pip3 install argcomplete
       $ activate-global-python-argcomplete
 
 Running Experiments
 -------------------
 
-As a simple example, we compare Insertion Sort and Bubble Sort on a set of instances
-(such an example is available in the  within `examples
-<https://github.com/hu-macsy/simexpal/tree/master/examples>`_).
-To this purpose, we created a new `sorting
-<https://github.com/hu-macsy/simexpal/tree/master/examples/sorting>`_
-directory, and wrote a short Python script
-`sort.py
-<https://github.com/hu-macsy/simexpal/blob/master/examples/sorting/sort.py>`_
-where we implemented the two algorithms.
-``sort.py`` accepts two arguments: the algorithm name (i.e. insertion-sort or bubble-sort)
-and the path to the instance.
-Then, you generated a bunch of instances and placed them in a directory called "instances"
-within "project" (in this example we just deal with a single instance
-`random_500.list
-<https://github.com/hu-macsy/simexpal/blob/master/examples/sorting/instances/random_500.list>`_).
-Now we can run a sorting algorithm on a specific instance with:
+Let us now take a closer look at running experiments. As an example, we compare
+the sorting algorithms *insertion sort* and *bubble sort* using a set of
+instances. Please find the resources in our `example folder
+<https://github.com/hu-macsy/simexpal/tree/master/examples>`_. 
+
+If you look at the `sorting directory
+<https://github.com/hu-macsy/simexpal/tree/master/examples/sorting>`_ in our
+examples, you will find `my_sort.py
+<https://github.com/hu-macsy/simexpal/blob/master/examples/sorting/my_sort.py>`_
+including an implementation of the two algorithms *insertion sort* and *bubble
+sort*. 
+
+``my_sort.py`` expects two arguments: 
+
+1. The algorithm name (i.e. *insertion-sort* or *bubble-sort*). 
+2. The path to the instance. 
+
+In the above example, we generated a bunch of instances which were written to
+the *instances* folder at root level where the ``experiments.yml`` file is
+present. 
+
+For this example, we generate a new instance using the ``generate.py`` script.
+First we navigate to the ``sorting`` folder:
+
+.. code-block:: bash
+
+  $ cd examples/sorting
+
+Then we generate a new instance file which contains 500 randomly generated
+integers in list form with one integer per line:
+
+.. code-block:: bash
+
+  $ python3 generate.py -o ./instances/random-500 500
+
+Now we can run *insertion-sort* on the newly generate instance: 
```

Comment:
I think there should be commas here: "First, we navigate ... . Then, we generate ... . Now, we can run ..."

Reply:
Good point. I rephrased those enumerations.